src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.*;
```

Comment:
Two things here: 
1. Keep everything in a package
2. Only import things that are needed to be used. Avoid using wildcard imports